[ES] Update `Modding` (#8054)
ppy/osu-wiki: wiki/Modding/es.md

# Modding

> *Para otros usos, vea [Mod (desambiguación)](/wiki/Disambiguation/Mod).*

Reviewer: ```suggestion
*Para otros usos, véase [Mod (desambiguación)](/wiki/Disambiguation/Mod).*
```

> **Modding** es el acto de proveer retroalimentación al [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que está modeando es considerado como un *modder*, y sus posts colectivos en una discusión de un beatmap se denomina como un *mod*.

Reviewer: ```suggestion
**Modding** es el acto de proveer comentarios al [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que está modeando es considerado como un *modder*, y sus posts colectivos en una discusión de un beatmap se denomina como un *mod*.
```

> Los mods son principalmente usados para mejorar beatmaps que apuntan a un [estado Clasificado](/wiki/Beatmap/Category#clasificados). Debido a que es un proceso detallado y analitico, modear beatmaps también es una manera popular de mejorar el entendimiento del beatmapping.

Reviewer 2: ```suggestion
Los mods son principalmente usados para mejorar beatmaps que apuntan a un estado [clasificado](/wiki/Beatmap/Category#clasificados). Debido a que es un proceso detallado y analítico, modear mapas también es una manera popular de mejorar el entendimiento del beatmapping.
```

Reviewer: No lo veo así en la versión en inglés

Reviewer: No lo veo así en la versión en inglés

Reviewer 2: Pues si, pero cuando entro al link, me lleva a una sección donde solo se coloco "ranked" en una pagina de categorías, no se si se debería mantener el cambio o lo dimito?

> Los mods son principalmente usados para mejorar beatmaps que apuntan a un [estado Clasificado](/wiki/Beatmap/Category#estatus-de-rankeo). Debido a que es un proceso detallado y analitico, modear beatmaps también es una manera popular de mejorar el entendimiento del beatmapping.

Reviewer: ```suggestion
Los mods son principalmente usados para mejorar beatmaps que apuntan a un [estado Clasificado](/wiki/Beatmap/Category#clasificados). Debido a que es un proceso detallado y analitico, modear beatmaps también es una manera popular de mejorar el entendimiento del beatmapping.
```

> ## Encontrando beatmaps para modear

Reviewer 2: ```suggestion
## Encontrando mapas para modear
```

Reviewer: Beatmap no se traduce

Mientras hay incontables maneras de descubrir beatmaps para modear, los siguientes son algunos métodos comunes:

> - La [Modding queue forum](https://osu.ppy.sh/community/forums/60) es la plataforma usada más activa para encontrar beatmaps para modear y para solicitar mods. Un usuario puede crear su propio hilo, refiriéndose como una *queue*, para que otros usuarios publiquen beatmaps que quieran que sean modeados.

Reviewer: ```suggestion
- El [foro de Modding Queues](https://osu.ppy.sh/community/forums/60) es la plataforma usada más activa para encontrar beatmaps para modear y para solicitar mods. Un usuario puede crear su propio hilo, refiriéndose como una *queue*, para que otros usuarios publiquen beatmaps que quieran que sean modeados.
```

  El creador de cada queue dicta las reglas de cola, controlando que beatmaps son viables para recibir mods. Las colas usualmente abren por solicitudes de dos tipos de mods:
  
  - **Normal mod (NM):** Un mod que será proporcionado sin recibir trabajo alguno por parte del beatmaper solicitante.
  - **Mod for Mod (M4M):** Un intercambio que requiere que cada usuario modee el beatmap del otro.

- Los modders reconocidos a menudo anotan en sus perfiles de usuario si están o no aceptando solicitudes de mods.

> - Buscando el listado de beatmaps pendientes como beatmaps para modear **no** son recomendados debido a que la mayoría de envíos no están apuntando al estado Ranked, por lo que sus creadores no responderán potencialmente por mods.

Reviewer: ```suggestion
- Buscando el listado de beatmaps pendientes como beatmaps para modear **no** son recomendados debido a que la mayoría de envíos no están apuntando al estado Clasificado, por lo que sus creadores no responderán potencialmente por mods.
```

## Como modear

> Para modear un beatmap, empieza descargando la versión más reciente y abre su [página de discusión](/wiki/Beatmap_discussion). Juega la [dificultad](/wiki/Beatmap/Difficulty) que quieras modear y ten en cuenta cualquier problema relacionado con el gameplay. Cuando finalices, abre la dificultad en el editor y analízala para encontrar otros errores.

Reviewer: ```suggestion
Para modear un beatmap, empieza descargando la versión más reciente y abre su [página de discusión](/wiki/Beatmap_discussion). Juega la [dificultad](/wiki/Beatmap/Difficulty) que quieras modear y ten en cuenta cualquier problema relacionado con el juego. Cuando finalices, abre la dificultad en el editor y analízala para encontrar otros errores.
```

> Cuando te encuentres con un problema, selecciona los hit objects que sean relevantes, luego usa el atajo de copia (`Ctrl` + `C`) para copiar la información de la línea de tiempo como texto. En la discusión del beatmap, pega (`Ctrl` + `V`) cualquier línea de tiempo aplicable y tu retroalimentación. La pestaña de la `linea del tiempo` se usa para problemas con respecto a partes específicas de un beatmap, la pestaña `General (Esta dificultad)` se usa para dar retroalimentación en secciones largas de un beatmap, y la pestaña `General (Todas las dificultades)` se usa para problemas que se apliquen con todas las dificultades de un beatmap.

Reviewer: ```suggestion
Cuando te encuentres con un problema, selecciona los objetos de golpeo que sean relevantes, luego usa el atajo de copiar (`Ctrl` + `C`) para copiar la información de la línea de tiempo como texto. En la discusión del beatmap, pega (`Ctrl` + `V`) cualquier línea de tiempo aplicable y tu comentario. La pestaña `Línea de tiempo` se usa para problemas con respecto a partes específicas de un beatmap, la pestaña `General (Esta dificultad)` se usa para dar comentarios en secciones largas de un beatmap, y la pestaña `General (Todas las dificultades)` se usa para problemas que se apliquen con todas las dificultades de un beatmap.
```

> Todos los mod posts deben abordar cuál es el problema, porque es un problema, y como arreglarlo. Revise el [Código de conducta para modear y mapear](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) para más detalles en como dar formato a los mod posts.

Reviewer: ```suggestion
Todos los mod posts deben abordar cuál es el problema, porque es un problema, y como arreglarlo. Revise el [Código de conducta para Modding y Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#haciendo-un-mod-post) para más detalles en cómo dar formato a los mod posts.
```

> Todos los mod posts deben abordar cuál es el problema, porque es un problema, y como arreglarlo. Revise el [Código de conducta para Modding y Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) para más detalles en cómo dar formato a los mod posts.

Reviewer: ```suggestion
Todos los mod posts deben abordar cuál es el problema, porque es un problema, y como arreglarlo. Revise el [Código de conducta para Modding y Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#haciendo-un-mod-post) para más detalles en cómo dar formato a los mod posts.
```

> Los beatmappers regularmente ofrecen [kudosu](/wiki/Modding/Kudosu) a los mod posts que ellos consideren como útiles al cliquear el botón de *pulgar arriba*. Esta estadística es rastreada por medio del perfil del usuario y se toma en cuenta al aplicar para ingresar a los [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators).

Reviewer: ```suggestion
Los beatmappers regularmente ofrecen [kudosu](/wiki/Modding/Kudosu) a los mod posts que ellos consideren como útiles al cliquear el botón de votar *pulgar arriba*. Esta estadística es rastreada por medio del perfil del usuario y se toma en cuenta al aplicar para ingresar a los [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators).
```

## Tipos de mod posts

> El objetivo de cada publicación de un `Problema` o una `Sugerencia` en una página de discusión de beatmap es explicar todo lo relacionado con un beatmap en el contexto de su diseño.

Reviewer: ```suggestion
El objetivo de cada post de un `Problema` o una `Sugerencia` en una página de discusión de beatmap es explicar todo lo relacionado con un beatmap en el contexto de su diseño.
```

Muchos mod posts abordan errores en un beatmap. Por ejemplo, un clap hitsound faltante que se esperaría por el patrón de hitsounding del beatmap es probable que sea un problema.

> Proveer maneras alternativas de mapear es otro tipo común de mod posts. Mientras parte del beatmap pueda ser aceptable en su forma actual, un modder puede explicar como mejorarlo aun más diseñándolo de manera distinta.

Reviewer: ```suggestion
Proveer maneras alternativas de beatmapping es otro tipo común de mod posts. Mientras parte del beatmap pueda ser aceptable en su forma actual, un modder puede explicar como mejorarlo aún más diseñándolo de manera distinta.
```

> # Modding (Español)

Reviewer 3: ```suggestion
# Modding
```

Author: Done

*Para otros usos, revisar [Mod (desambiguación)](/wiki/Disambiguation/Mod).*

**Modding** es el acto de proveer retroalimentación al [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que está modeando es considerado como un *modder*, y sus posts colectivos en una discusión de un beatmap se denomina como un *mod*.

Los Mods son principalmente usados para mejorar beatmaps que apuntan a un [estado Ranked](/wiki/Beatmap/Category#ranked). Debido a que es un proceso detallado y analitico, modear beatmaps tambien es una manera popular de mejorar el entendimiento del beatmapping.

## Encontrando beatmaps para modear

Mientras hay incontables maneras de descubrir beatmaps para modear, los siguientes son algunos métodos comunes:

> - La [cola del foro de modding](https://osu.ppy.sh/community/forums/60) es la plataforma usada más activa para encontrar beatmaps para modear y para solicitar mods. Un usuario puede crear su propio hilo, refiriéndose como una *cola*, para que otros usuarios publiquen beatmaps que quieran que sean modeados.

Reviewer 3: Could you reformulate this sentance to keep `[Modding queue](https://osu.ppy.sh/community/forums/60)` in english

Author: Done

  El creador de cada cola dicta las reglas de cola, controlando que beatmaps son viables para recibir mods. Las colas usualmente abren por solicitudes de dos tipos de mods:
  
  - **Normal mod (NM):** Un mod que será proporcionado sin recibir trabajo alguno por parte del beatmaper solicitante.
  - **Mod for Mod (M4M):** Un intercambio que requiere que cada usuario modee el beatmap del otro.

- Los modders reconocidos a menudo anotan en sus perfiles de usuario si están o no aceptando solicitudes de mods.

- Buscando el listado de beatmaps pendientes como beatmaps para modear **no** son recomendados debido a que la mayoría de envíos no están apuntando al estado Ranked, por lo que sus creadores no responderán potencialmente por mods.

## Como modear

Para modear un beatmap, empieza descargando la versión más reciente y abre su [página de discusión](/wiki/Beatmap_discussion). Juega la [dificultad](/wiki/Beatmap/Difficulty) que quieras modear y ten en cuenta cualquier problema relacionado con el gameplay. Cuando finalices, abre la dificultad en el editor y analízala para encontrar otros errores.

> Cuando te encuentres con un problema, selecciona los hit objects que sean relevantes, luego usa el atajo de copia (`Ctrl` + `C`) para copiar la información de la línea de tiempo como texto. En la discusión del beatmap, pega (`Ctrl` + `V`) cualquier línea de tiempo aplicable y tu retroalimentación.

Reviewer 3: **Missing all this sentance _(see en.md file)_:**
The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties of a beatmap.

Author: Added.
